x/auth/vesting/types/vesting_account.go

```diff
@@ -556,17 +553,18 @@ func (pva PeriodicVestingAccount) GetVestingCoins(blockTime time.Time) sdk.Coins
 	return pva.OriginalVesting.Sub(pva.GetVestedCoins(blockTime))
 }
 
-// SpendableCoins returns the total number of spendable coins per denom for a
-// periodic vesting account.
-func (pva PeriodicVestingAccount) SpendableCoins(blockTime time.Time) sdk.Coins {
-	return pva.BaseVestingAccount.SpendableCoinsVestingAccount(pva.GetVestingCoins(blockTime))
+// LockedCoins returns the set of coins that are not spendable (i.e. locked)
+// given the current account balance, only including tokens of vesting
+// denominations, and the current block time.
+func (pva PeriodicVestingAccount) LockedCoins(balance sdk.Coins, blockTime time.Time) sdk.Coins {
```

Comment:
Maybe a note here about how to find liquid coins using this?

Reply:
Absolutely.

Reply:
done.